prime-dotnet-webapi/Configuration/UserClauseConfiguration.cs

```diff
@@ -10,24 +10,32 @@ public class UserClauseConfiguration : SeededTable<UserClause>
         private string ruClause2;
         private string oboClause1;
         private string oboClause2;
+        private string oboClause3;
```

Comment:
Ugh!  Hopefully TOA versioning is prioritized in the near future

Reply:
agreeeed